### PR TITLE
docs(README): add tsserver part

### PR DESCRIPTION
**Pull request by kud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://travis-ci.org/theia-ide/typescript-language-server.svg?branch=master)](https://travis-ci.org/theia-ide/typescript-language-server)
 
 # TypeScript Language Server
-[Language Server Protocol](https://github.com/Microsoft/language-server-protocol) implementation for TypeScript wrapping `tsserver`.
+[Language Server Protocol](https://github.com/Microsoft/language-server-protocol) implementation for TypeScript wrapping `tsserver`. You must install it first via `$ npm install -g typescript`.
```
Comment (akosyakov) [inline README.md]: Not necessary, it can come with multiple places, i.e. your project can declare it as a dev dependency.

Comment (kud) [reply README.md]: I understand, yeah. I should just precise you need typescript in any way?

**Comment by DonnieWest** [comment]
@kud if you're still interested, I'd love some instructions in the README for `tsserver` installation. It would be necessary to enumerate where the project resolves `tsserver` though (`package.json`, globally, etc)

File: README.md (hunk 1 above)
Comment (apexskier) [reply README.md:5]: I also think this shouldn't be added. Installing globally is [officially considered bad practice](https://nodejs.org/en/blog/npm/npm-1-0-global-vs-local-installation/#which-to-choose).

Instead, I'd suggest linking to the `tsserver` documentation, which has more info about how to get and use it.

```suggestion
[Language Server Protocol](https://github.com/Microsoft/language-server-protocol) implementation for TypeScript wrapping [`tsserver`](https://github.com/Microsoft/TypeScript/wiki/Standalone-Server-%28tsserver%29).
```

**Comment by DonnieWest** [comment]
No response, closing for now

**Pull request closed without merging**